src/Engine/ProtoCore/Lang/CallSite.cs

```diff
@@ -1155,9 +1169,9 @@ private FunctionGroup FirstFunctionGroupInInheritanceChain(RuntimeCore runtimeCo
                 {
                     if (fep.FormalParams[i].rank == Constants.kArbitraryRank)
                         noArbitraries++;
```

Comment:
I really hate this variable name - kept reading it as No Arbitraries, and was very confused.

Reply:
renamed it